fhir-ig-carin-bb/src/test/java/com/ibm/fhir/ig/carin/bb/test/BBResourceProviderTest.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.ig.carin.bb.test;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.ig.carin.bb.BBResourceProvider;
+import com.ibm.fhir.registry.spi.FHIRRegistryResourceProvider;
+
+public class BBResourceProviderTest {
```

Comment:
maybe add a test to loop through all the resources and validate them?

Reply:
Added validation to unit test in US Core but left it out of CARIN BB (the validation in IndexGenerator is sufficient per our conversation).